grr/core/grr_response_core/lib/rdfvalues/osquery.py

```diff
@@ -99,6 +99,13 @@ class OsqueryResult(rdf_structs.RDFProtoStruct):
   protobuf = osquery_pb2.OsqueryResult
   rdf_deps = [OsqueryTable]
 
+  def GetTableColumns(self):
+    return [column.name for column in self.table.header.columns]
+  
+  def GetTableRows(self):
+    return [row.values for row in self.table.rows]
```

Comment:
Prefer to use type annotations in all newly written code. Also, do you need to return a list? Wouldn't iterator suffice and let the caller decide whether to allocate a list out of it?